Solutions/Marain.UserNotifications.Client/Marain/UserNotifications/Client/ApiDeliveryChannel/Resources/NotificationResource.cs

```diff
@@ -98,7 +97,7 @@ public DateTimeOffset Timestamp
             get
             {
                 if (this.HalDocument.TryGetProperty("timestamp", out JsonElement timestamp) &&
-                    timestamp.ValueKind == JsonValueKind.Object)
+                    timestamp.ValueKind == JsonValueKind.String)
                 {
                     return DateTimeOffset.Parse(timestamp.GetString());
```

Comment:
Any reason not to use `JsonElement.GetDateTimeOffset()`? https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.getdatetimeoffset?view=netcore-3.1

Reply:
Nope.